akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -58,9 +58,9 @@ abstract class JournalSpec(config: Config) extends PluginSpec(config) {
   def writeMessages(fromSnr: Int, toSnr: Int, pid: String, sender: ActorRef, writerUuid: String): Unit = {
     val msgs =
       if (supportsAtomicPersistAllOfSeveralEvents)
```

Comment:
I think those two if-else branches were accidentally interchanged. It looks like the test data for `persistAll` is located in the else branch, but it should be here.